### PR TITLE
Polish & delabs `Exploring public spaces` feature

### PR DESCRIPTION
**Pull request by t3chguy**
Requires https://github.com/vector-im/element-web/pull/25991

image
image
image

---
Here's what your changelog entry will look like:

## ✨ Features
 * Polish & delabs `Exploring public spaces` feature ([\#11423](https://github.com/matrix-org/matrix-react-sdk/pull/11423)).

### CODE REVIEW

File: res/css/views/dialogs/_SpotlightDialog.pcss [hunk 1]
```diff
@@ -97,7 +97,11 @@ limitations under the License.
             }
 
             &.mx_SpotlightDialog_filterPublicRooms::before {
-                mask-image: url("$(res)/img/element-icons/roomlist/explore.svg");
+                mask-image: url("$(res)/img/element-icons/roomlist/hash-circle.svg");
+            }
+
+            &.mx_SpotlightDialog_filterPublicSpaces::before {
+                mask-image: url("$(res)/img/element-icons/spaces.svg");
             }
```
Comment (germain-gg) [inline res/css/views/dialogs/_SpotlightDialog.pcss:105]: I believe we're trying to favour using SVG react elements over `mask-image` 
https://github.com/matrix-org/matrix-react-sdk/blob/develop/docs/icons.md#icons

**Review by germain-gg — CHANGES_REQUESTED** [review]
CI is slightly, but by large looks pretty good

File: res/css/views/dialogs/_SpotlightDialog.pcss (hunk 1 above)
Comment (t3chguy) [reply res/css/views/dialogs/_SpotlightDialog.pcss:105]: Agreed but I think matching the surrounding code is more important given all these icons will need changing as part of the compound work anyway

File: res/img/element-icons/spaces.svg [hunk 2]
```diff
@@ -0,0 +1,3 @@
...
...7.41682 11.5909H10.2814C10.9142 11.5909 11.4272 11.0779 11.4272 10.4451V7.58052C11.4272 6.9477 10.9142 6.43469 10.2814 6.43469H7.41682ZM7.41682 13.9462L10.2814 13.9462L10.2814 16.8108L7.41682 16.8108L7.41682 13.9462ZM6.271 13.9462C6.271 13.3134 6.784 12.8004 7.41682 12.8004H10.2814C10.9142 12.8004 11.4272 13.3134 11.4272 13.9462V16.8108C11.4272 17.4436 10.9142 17.9566 10.2814 17.9566H7.41682C6.784 17.9566 6.271 17.4436 6.271 16.8108V13.9462ZM16.6471 7.58052L13.7825 7.58052L13.7825 10.4451L16.6471 10.4451L16.6471 7.58052ZM13.7825 6.43469C13.1497 6.43469 12.6367 6.94769 12.6367 7.58052V10.4451C12.6367 11.0779 13.1497 11.5909 13.7825 11.5909H16.6471C17.2799 11.5909 17.7929 11.0779 17.7929 10.4451V7.58052C17.7929 6.9477 17.2799 6.43469 16.6471 6.43469H13.7825ZM15.2148 16.8108C14.4238 16.8108 13.7825 16.1695 13.7825 15.3785C13.7825 14.5875 14.4238 13.9462 15.2148 13.9462C16.0058 13.9462 16.6471 14.5875 16.6471 15.3785C16.6471 16.1695 16.0058 16.8108 15.2148 16.8108ZM12.6367 15.3785C12.6367 13.9546 13.7909 12.8004 15.2148 12.8004C16.6386 12.8004 17.7929 13.9546 17.7929 15.3785C17.7929 16.8023 16.6386 17.9566 15.2148 17.9566C13.7909 17.9566 12.6367 16.8023 12.6367 15.3785Z" fill="black"/>
```
[the remark this answers is not in the document]
Comment (t3chguy) [reply res/img/element-icons/spaces.svg:2]: This matches the other icons in the area, would rather not have an inconsistent implementation which is more likely to lead to bugs

[the remark this answers is not in the document]
Comment (germain-gg) [reply res/img/element-icons/spaces.svg:2]: I believe we can safely commit to changing `fill="black"` to `fill="currentColor"` at the very least

**Review by germain-gg — APPROVED** [review]
Approving, but would like to see the `fill="currentColor"` included as it matches what is written in documentation is is what we said we would commit to doing

**Comment by daniellekirkwood** [comment]
The screenshots in the description are not representative of the experience I have in nightly - what am I doing wrong?

Or does this PR do more than de-labs?

**Comment by daniellekirkwood** [comment]
Ok - I tried the netlify build and played around with that. I can see where the screenshots come from and much prefer the netlify experience but that means the PR isn't quite accurtely named ?

**Comment by t3chguy** [comment]
> Or does this PR do more than de-labs?

It does indeed do more than de-labs, updated the PR title

**Review by daniellekirkwood — APPROVED** [review]
thanks for changing the title

**Pull request merged**